sponsors.md

```diff
@@ -49,6 +35,18 @@ Thank you to dm, for their many ongoing contributions to the codebase.
 <div class="lead text-left"><a href="https://fastly.com"><img src="/static/images/sponsors/fastly.png" height="125px"/></a></div>
 Thank you to Fastly, who provides the CDN for our downloads and repositories.
 
+### OSUOSL
+<div class="lead text-left"><a href="https://osuosl.org"><img src="/static/images/sponsors/osuosl.png" height="125px"/></a></div>
+Thank you to OSUOSL, who provides infrastructure to power our continuous integration system and automated testing.
+
+### CentOS CI
```

Comment:
I wonder if we should thank CentOS CI or CentOS or Fedora Infra.

Reply:
I debated just CentOS since that is the wiki/website/name/mailing list.